src/main/java/com/redhat/devtools/lsp4ij/settings/LanguageServerView.java

```diff
@@ -251,6 +255,7 @@ public void apply() {
                             launch,
                             getDisplayName(),
                             getCommandLine(),
+                            getEnvData().getEnvs(),

```

Comment:
I think we need to pass the whole getEnvData() object, that determines whether to use system env variables

Reply:
We don't need that since in your last PR you hard code that.